electron/index.ts

```diff
@@ -188,7 +189,7 @@ myPromiseIpc.on('is-reachable', (config: cordova.plugins.outline.ServerConfig) =
 function startProxying(config: cordova.plugins.outline.ServerConfig, id: string) {
   return process_manager.teardownProxy()
       .catch((e) => {
-        console.error('error tearing down current proxy', e);
+        SentryLogger.error(`error tearing down current proxy: ${e}`);
```

Comment:
Can't that leak PII like the IP address?

Reply:
I don't think so, the system proxy and the HTTP proxy are configured with the local IP address.